hive/test/tests/box/box_base_test.dart

```diff
@@ -12,11 +12,11 @@ import '../common.dart';
 
 import '../mocks.dart';
 
-class _FakeStorageBackend extends Fake implements StorageBackend {}
+//class _FakeStorageBackend extends Fake implements StorageBackend {}
```

Comment:
I would suggest you to remove those classes instead of commenting

Reply:
Ok, I thought that they could be useful in the future. But I'm deleting them.

Reply:
Done!